lib/rspec/abq/extensions.rb

```diff
@@ -87,9 +87,10 @@ def run_with_abq(reporter)
             # be sent to us from ABQ, we now loop over all the examples, and mark
             # every one that we must run in this group as a failure.
             for_filtered_examples(reporter) do |example|
-              next unless Abq.target_test_case.is_example?(example)
-
-              example.fail_with_exception(abq_reporter, ex)
```

Comment:
There were two issues here:
* `abq_reporter` was undefined
* we were looping over all of the examples but only marking the one currently assigned to the worker as failed. This causes a problem if the worker pulls another spec in the example group, but we've already moved past that in iterating over the tests